lib/internal/modules/esm/formats.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
```

Comment:
i forget exactly what but there was some oddity that made me move it.

Reply:
That’s fine, but we can still do this reorganization in a tiny little PR on its own, and then this big one can follow.

Reply:
I think splitting then rebasing would be more work.

Reply:
This was me. `mimeToFormat` was introduced in this PR, and seemed primed to be a shared utility. I moved the other pieces directly relevant to this here too since it seemed part of the same atomic change.